dev/ant_build.js/build.gradle

```diff
@@ -0,0 +1,144 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+import groovy.json.JsonSlurper
+
+configurations {
+    libs
+    libs1
+    libs2
+    libs3
+    libs4
+    libs5
+    libs6
+    libs7
+    libs8
+    libs9
+    libs10
+}
+
+dependencies {
+    libs "com.fasterxml.jackson.core:jackson-annotations:2.2.3"
```

Comment:
Similar to my comment above. I'd rather not pull in specific versions here. Instead, use a common bundle and use the same version across Open Liberty. In this case you would put `libs project(':io.openliberty.com.fasterxml.jackson')` to pull in our existing bundle that includes version 2.11.2.

Reply:
Same applies to any others that are also used elsewhere in Open Liberty. Not quite as important if this is the only place we depend on something.

Reply:
after talking to Katie, we will address this in #15902

Reply:
@cbridgha Should this work as it is? Version 2.2.3 is not listed in the oss_dependencies.maven file. I am surprised that this feature did not need to add anything to those maven files!